libs/components/data-manager/testing/src/modules/data-manager/data-manager-toolbar-harness.ts

```diff
@@ -0,0 +1,259 @@
+  ): Promise<SkyDataManagerToolbarLeftItemHarness> {
+    return await this.locatorFor(
+      SkyDataManagerToolbarLeftItemHarness.with(filter),
+    )();
+  }
+
+  /**
+   * Gets an array of all data manager toolbar left items.
+   */
+  public async getLeftItems(
+    filters?: SkyDataManagerToolbarLeftItemHarnessFilters,
+  ): Promise<SkyDataManagerToolbarLeftItemHarness[]> {
+    const items = await this.locatorForAll(
+      SkyDataManagerToolbarLeftItemHarness.with(filters || {}),
+    )();
+
+    if (filters && items.length === 0) {
+      throw new Error(
+        `Unable to find any data manager toolbar left items with filter(s): ${JSON.stringify(filters)}`,
+      );
+    }
+
+    return items;
+  }
+
+  /**
+   * Gets a harness for the only show selected checkbox.
+   */
+  public async getOnlyShowSelected(): Promise<SkyCheckboxHarness | null> {
+    return await this.locatorForOptional(
+      SkyCheckboxHarness.with({
+        ancestor:
+          'sky-toolbar-view-actions.sky-data-manager-only-show-selected',
+      }),
+    )();
+  }
+
+  /**
+   * Gets a harness for a specific data manager toolbar primary item that meets certain criteria.
+   */
+  public async getPrimaryItem(
+    filter: SkyDataManagerToolbarPrimaryItemHarnessFilters,
+  ): Promise<SkyDataManagerToolbarPrimaryItemHarness> {
+    return await this.locatorFor(
+      SkyDataManagerToolbarPrimaryItemHarness.with(filter),
+    )();
+  }
+
+  /**
+   * Gets an array of all data manager toolbar primary items.
+   */
+  public async getPrimaryItems(
+    filters?: SkyDataManagerToolbarPrimaryItemHarnessFilters,
+  ): Promise<SkyDataManagerToolbarPrimaryItemHarness[]> {
+    const items = await this.locatorForAll(
+      SkyDataManagerToolbarPrimaryItemHarness.with(filters || {}),
+    )();
+
+    if (filters && items.length === 0) {
+      throw new Error(
+        `Unable to find any data manager toolbar primary items with filter(s): ${JSON.stringify(filters)}`,
+      );
+    }
+
+    return items;
+  }
+
+  /**
+   * Gets a harness for a specific data manager toolbar right item that meets certain criteria.
+   */
+  public async getRightItem(
+    filter: SkyDataManagerToolbarRightItemHarnessFilters,
+  ): Promise<SkyDataManagerToolbarRightItemHarness> {
+    return await this.locatorFor(
+      SkyDataManagerToolbarRightItemHarness.with(filter),
+    )();
+  }
+
+  /**
+   * Gets an array of all data manager toolbar right items.
+   */
+  public async getRightItems(
+    filters?: SkyDataManagerToolbarRightItemHarnessFilters,
+  ): Promise<SkyDataManagerToolbarRightItemHarness[]> {
+    const items = await this.locatorForAll(
+      SkyDataManagerToolbarRightItemHarness.with(filters || {}),
+    )();
+
+    if (filters && items.length === 0) {
+      throw new Error(
+        `Unable to find any data manager toolbar right items with filter(s): ${JSON.stringify(filters)}`,
+      );
+    }
+
+    return items;
+  }
+
+  /**
+   * Clicks the data manager search.
```

Comment:
these jsdocs dont seem to relate to the function

Reply:
fixed